rest/models.py

```diff
@@ -12,3 +13,64 @@ def __init__(self, response: requests.Response):
         dict_rsp = json.loads(response.content)
         self.buy = dict_rsp.get('buy')
         self.sell = dict_rsp.get('sell')
+
+class Ticker:
+    high: decimal.Decimal
+    last: decimal.Decimal
+    timestamp: str
+    bid: decimal.Decimal
+    vwap: decimal.Decimal
+    volume: decimal.Decimal
+    low: decimal.Decimal
+    ask: decimal.Decimal
+    open: decimal.Decimal
+
+    def __init__(self, response: requests.Response):
+        dict_rsp = json.loads(response.content)
+        self.high = dict_rsp.get('high')
+        self.last = dict_rsp.get('last')
+        self.timestamp = dict_rsp.get('timestamp')
+        self.bid = dict_rsp.get('bid')
+        self.vwap = dict_rsp.get('vwap')
+        self.volume = dict_rsp.get('volume')
+        self.low = dict_rsp.get('low')
+        self.ask = dict_rsp.get('ask')
+        self.open = dict_rsp.get('open')
+
+class HourlyTicker:
```

Comment:
This model is completely redundant. It's completely the same as `Ticker`. It doesn't matter whether is hourly or daily or weekly, it's still ticker.
Hourly ticker should also return Ticker model.

Reply:
Model removed, moved to Ticker only.
Resolving as I see no futher conflict.